docs/GETTING_STARTED.md

```diff
@@ -0,0 +1,278 @@
+Types](https://github.com/awslabs/soci-snapshotter/issues/42).  For now we are
+relying on the ORAS project for the referrers featre.
+
+Given this we want to start our registry using an OCI image from the ORAS 
+project.  We will run it on port 5000 as an HTTP (**NOT** HTTPS) server without
+certificates.  Since this is just a locally run HTTP server we can interact with
+it using standard `curl` commands to confirm our actions.
+To do this we will use Docker :
+
+```
+docker run -d -p 5000:5000 --restart=always --name registry ghcr.io/oras-project/registry:v1.0.0-rc
+```
+
+To confirm this is running lets run the following commands:
+
+```
+docker ps
+```
+which should yield a running container at 0.0.0.0:5000 for the ORAS image. Now
+to check that we can interact with the server run the following :
+```
+curl http://localhost:5000/v2/_catalog
+```
+This should return with `{"repositories":[]}` given that we haven't put anything
+in there yet
+
+## Push an image to the registry using ctr
+First things first, let's choose our favorite image.  For the purpose of this
+document we will use the latest image of rabbitmq from DockerHub, aptly named
+`docker.io/library/rabbitmq:latest`.
+
+To do this with ctr use the following command :
+```
+ctr i pull docker.io/library/rabbitmq:latest
+```
+To confirm this image (or the image you have chosen) is loaded into ctr please
+run :
+```
+ctr i ls
+```
+and you should see the image.  Now we need to add an image reference it so that
+it can be added to our repository.
+```
+ctr i tag docker.io/library/rabbitmq:latest localhost:5000/rabbitmq:latest
+```
+Note that the `localhost:5000` is the address of our registry, `rabbitmq` will
+be the name of the repository, and `latest` will be the tag within the
+repository.
+
+To confirm this worked once more run `ctr i ls` and see that there is now a new
+line with the tag you just created.  Also confirm that the digest, the long
+alphanumeric hash that is prefixed by "sha256:", matches the base image.
+
+Finally, we can push to the local registry:
+
+```
+ctr i push --platform linux/amd64 --plain-http localhost:5000/rabbitmq:latest
+```
+
+**NOTE** : the platform tag might be different depending what environment you
+are working in.
+
+Once again let's confirm via a curl command :
+```
+curl http://localhost:5000/v2/_catalog
+```
+We should now see that our repository has been created
+`{"repositories":["rabbitmq"]}`.  We can further inspect and see what tags were
+created (you may need to change 'rabbitmq' if you named your repo something else).
+```
+curl http://localhost:5000/v2/rabbitmq/tags/list
+```
+We can also look at the details of the tag using (you may need to change
+'rabbitmq' and 'latest' if you named your repo and tag something else) :
+```
+curl http://localhost:5000/v2/rabbitmq/manifests/latest
+```
+## Create and push a SOCI index
+Now that we have confidence that our registry contains the image let's work on
+the SOCI index.
```

Comment:
Should have a brief explanation about SOCI index or have a link to https://github.com/awslabs/soci-snapshotter#no-image-conversion.